src/app.py

```diff
@@ -4,56 +4,96 @@
 from dash import Dash, dcc, callback, Output, Input, html
 import plotly.express as px
 
-
 # Initiatlize the app
 app = Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
 server = app.server
 
-crime_data = pd.read_csv("data/processed/crimedata_processed.csv")
+crime_df = pd.read_csv("data/processed/crimedata_processed.csv")
+crime_df['DATE'] = pd.to_datetime(crime_df[['YEAR','MONTH','DAY','HOUR','MINUTE']])
+crime_df.set_index('DATE',inplace=True)
+
+# Create dropdown options
+crime_type_options = [{'label': crime_type, 'value': crime_type} for crime_type in crime_df['TYPE'].unique()]
+crime_type_options.insert(0, {'label': 'All', 'value': 'All'})
+neighbourhood_options = [{'label': neighbourhood, 'value': neighbourhood} for neighbourhood in crime_df['NEIGHBOURHOOD'].unique()]
+neighbourhood_options.insert(0, {'label': 'All', 'value': 'All'})
 
 app.layout = dbc.Container([
     dbc.Row([
-        html.H1('VANCOUVER CRIME TRACKER 2023'),
+        html.H1('VANCOUVER CRIME TRACKER 2023', style={'color': 'purple'}),
         html.Br(),
     ]),
     dbc.Row([
         dbc.Col([
-            html.H1('Total Crime Count'),
-        html.Br(),
+            dbc.Card([
+                dbc.CardBody([
+                    html.H3('Total Crime Count', style={'color': 'blue'}),
```

Comment:
what is the color we should stick to?

Reply:
I tried different colours but we can think about it!